client/components/likes.jsx

```diff
@@ -3,22 +3,42 @@ import React from 'react';
 class Likes extends React.Component {
   constructor(props) {
     super(props);
-    this.state = {};
+    this.state = {
+      liked: false,
+    };
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick() {
+    this.setState(prevState => ({
+      liked: !prevState.liked,
+    }));
   }
 
   render() {
+    /* * * * * * Styling * * * * * */
     const likeStyle = {
       position: 'absolute',
       width: 70,
       height: 25,
       bottom: 30,
       left: 10,
-      border: '2px solid red',
-      borderRadius: '8px',
+      textAlign: 'center',
+      backgroundColor: 'white',
+      border: '1px solid lightgrey',
     };
+    /* * * * Conditional rendering * * * */
+    let page;
+    const { liked } = this.state;
+    if (liked === false) {
```

Comment:
Here a ternary operator might help for succinct code, since liked is undeniably a boolean value: 

liked ? page = redheart : page = blackheart